scripts/sample_export_config.json

```diff
@@ -113,6 +113,26 @@
       "name": "sodium_100g",
       "nutrient_id": 1093,
       "scale": 0.001
-    }
+    },
+    {
+      "name": "calcium_100g",
+      "nutrient_id": 1087,
+      "scale": 0.001
+    },
+    {
+      "name": "iron_100g",
+      "nutrient_id": 1089,
+      "scale": 0.001
+    },
+    {
+      "name": "vitamin-a_IU_100g",
+      "nutrient_id": 1089,
+      "scale": 1
+    },
+    {
+      "name": "vitamin-C_IU_100g",
+      "nutrient_id": 1089,
```

Comment:
Same here.  In this case there is actually no Vitamin C in IU units, only in mg.

Sometimes the same vitamin appears multiple times in the nutrients definition file (`nutrient.csv`).  In these cases it's helpful to run the `summarize` command and look at the file that is also called `nutrient.csv` in the output of `summarize`.  This gives the nutrients that are actually used in the Branded Food dataset with their frequencies.  This is useful for determining which nutrients to prioritize.